charm/bundle/bundles.yaml

```diff
@@ -0,0 +1,21 @@
+calico-testing-formation:
+  services:
+    etcd:
+      charm: cs:~kubernetes/trusty/etcd
+      num_units: 1
+    docker:
+      charm: cs:trusty/docker
+      num_units: 2
+      options:
+        latest: true
+        version: 1.6.2
+    calico-docker:
+      charm: local:trusty/calico-docker
+      options:
+        docker-image: calico/node:latest
```

Comment:
This should be tied to releases rather than the auto builds from master. It's another thing to update and test when we do a release, but worth it as master can be unstable at times.